ci-operator/step-registry/servicemesh/kiali/cypress-tests/execute/servicemesh-kiali-cypress-tests-execute-commands.sh

```diff
@@ -1,18 +1,38 @@
-#!/bin/bash
-
 set -o nounset
 set -o errexit
 set -o pipefail
 
 CONSOLE_URL=$(cat $SHARED_DIR/console.url)
+export CONSOLE_URL
 OCP_API_URL="https://api.${CONSOLE_URL#"https://console-openshift-console.apps."}:6443"
-OCP_CRED_USR="kubeadmin"
-OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+export OCP_API_URL
+
+# login for interop
+if test -f ${SHARED_DIR}/kubeadmin-password
+then
+  OCP_CRED_USR="kubeadmin"
+  OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+  oc login ${OCP_API_URL} --username=${OCP_CRED_USR} --password=${OCP_CRED_PSW} --insecure-skip-tls-verify=true
+<<<<<<< HEAD
```

Comment:
This is a merge conflict. You need to resolve git merge conflicts

Reply:
I am reabsing it